__tests__/app.test.js

```diff
@@ -127,29 +128,74 @@ describe("GET /api/articles", () => {
     });
 });
 
-describe("error tests", () => {
-    test("404: responds with 404 when non-existent path when endpoint is spelt incorrectly", () => {
+describe("GET /api/articles/:article_id/comments", () => {
+    test("status 200, responds with an array of comments with the needed properties", () => {
+        return request(app)
+        .get("/api/articles/3/comments")
+        .then(({ body }) => {
+            const expected = {
+                comment_id: expect.any(Number),
+                votes: expect.any(Number),
+                created_at: expect.any(String),
+                author: expect.any(String),
+                body: expect.any(String),
+                article_id: 3
+            }
+            expect(body.comments).toBeInstanceOf(Array);
+            expect(body.comments[0]).toEqual(expected);
```

Comment:
We need to ensure that every element in the response array comes back looking as we expected with the correct article_id and everything else, it's also important to check the length of the response too.